index.html

```diff
@@ -0,0 +1,279 @@
+              <h3>Tea break</h3>
+            </div>
+            <p>
+              Lorem ipsum dolor sit amet consectetur adipisicing elit. Fugit
+              mollitia iust
+            </p>
+          </div>
+          <div class="program-content">
+            <div class="image-wrap">
+              <img
+                class="solo-content"
+                src="./assets/images/prayer.jpg"
+                alt="lecture"
+              />
+            </div>
+            <div class="program">
+              <h3>Isha Prayer</h3>
+            </div>
+            <p>
+              Lorem ipsum dolor sit amet consectetur adipisicing elit. Fugit
+              mollitia iust
+            </p>
+          </div>
+          <div class="program-content">
+            <div class="image-wrap">
+              <img
+                class="solo-content"
+                src="./assets/images/q&a.png"
+                alt="lecture"
+              />
+            </div>
+            <div class="program">
+              <h3>Q&A</h3>
+            </div>
+            <p>
+              Lorem ipsum dolor sit amet consectetur adipisicing elit. Fugit
+              mollitia iust
+            </p>
+          </div>
+
+          <div class="program-btn">
+            <button type="button" class="join btn">
+              Join gathering this friday!
+            </button>
+          </div>
+        </div>
+        <div class="see-program">
+          <a href="#">SEE THE WHOLE PROGRAM</a>
+        </div>
+      </section>
+    </section>
+    <!-- Main Program End -->
+
+    <!-- Featured Speakers Strats -->
+    <article id="featured-speakers" class="featured-speakers">
+      <section class="speaker-intro">
+        <h2>Featured Speakers</h2>
+        <hr class="speaker-intro-border" />
+        <!-- <div class="speaker-intro-border"></div> -->
+      </section>
+      <div id="speaker-wrapper" class="speaker-wrapper"></div>
+    </article>
+    <!-- Featured Speakers Ends -->
+    <!-- Partner starts -->
+    <article class="partner">
+      <section class="partner-intro">
+        <h2>Partner</h2>
+        <div class="partner-intro-border"></div>
+      </section>
+      <section class="partner-logo">
+        <img src="./assets/images/mozilla-1.svg" alt="mozilla" />
+        <img src="./assets/images/google.svg" alt="google" />
+        <img src="./assets/images/naver-1.svg" alt="naver" />
+        <img src="./assets/images/daumkakaologo-1.svg" alt="daumkakao" />
+        <img src="./assets/images/airbnb-1.svg" alt="airbnb" />
+      </section>
+    </article>
+    <!-- Partner Ends -->
+    <!-- Rights Starts -->
+    <footer class="rigts">
+      <img src="./assets/images/seminar-logo.png" alt="" />
+
+      <div class="rights-msg">
+        <h4>
+          © 2023 Islamic Dawah Center - “The believer’s shade on the Day of
+          Resurrection will be his charity.” (Al-Tirmidhi).
+        </h4>
+      </div>
+    </footer>
+    <!-- Rights Ends -->
```

Comment:
To have a complete website as the template, consider making the footer visible on the mobile view.